### DISCUSSION TITLE
Any difference using Steam vs ProtonUp-QT interface?

### DISCUSSION THREAD

**Discussion by UncleUncleRj** [DavidoTek/ProtonUp-Qt#411] [Q&A] [answered]
Hello - just was wondering if there was any difference using this interface vs the Steam properties window to choose the version of Proton I want (ie GE-Proton9-7) for a game, after ProtonUp-QT installs it?
I ask this because I've installed Red Alert 2 in Steam, but it doesn't show up in the ProtonUp-QT window, so I assume just selecting GE-Proton9-7 from the list on Steam takes care of everything...?

**Comment by DavidoTek** [comment, marked as answer]
Hi. There shouldn't be any difference between the tool selector in ProtonUp-Qt and Steam, it is just a convenience feature.
Please note that Steam needs to be closed when changing the compatibility tool using ProtonUp-Qt.

> I ask this because I've installed Red Alert 2 in Steam, but it doesn't show up in the ProtonUp-QT window

Do you mean it isn't displayed in the game list window?
That is strange. Normally, all games should be displayed there.

> I assume just selecting GE-Proton9-7 from the list on Steam takes care of everything...?

Yes, that should do it.

**Comment by sonic2kk** [comment]
About the Game List thing: I have had intermittent issues where games won't show up in the Games List, and then suddenly they do appear. From debugging, ProtonUp-Qt is not even reading the games from `appinfo.vdf` so I imagine Steam is occasionally not writing something correctly. It does not appear to be game-specific, and I also cannot consistently reproduce it, so I think it's something on the Steam side that occasionally isn't validated until the client is restarted.

Although again I don't believe this to be game-specific, I tried installing Red Alert 2 (I am not too familiar with the franchise, it was named "Command & Conquer Red Alert™ 2 and Yuri’s Revenge™" in my library) and I had to restart Steam for the installation status to be correctly reported.

![image](https://github.com/DavidoTek/ProtonUp-Qt/assets/7917345/c6aa14f4-8296-46f4-9997-3f0427bfe2c1)

You may have run into this issue.
